skyvern/forge/sdk/artifact/storage/s3.py

```
@@ -101,26 +102,42 @@ def build_script_file_uri(
             artifact_id=artifact.artifact_id,
             organization_id=artifact.organization_id,
-            uri=artifact.uri,
+            uri=uri,
             storage_class=sc,
             tags=tags,
         )
-        await self.async_client.upload_file(artifact.uri, data, storage_class=sc, tags=tags)
+        await self.async_client.upload_file(uri, data, storage_class=sc, tags=tags)
 
-    async def _get_storage_class_for_org(self, organization_id: str, bucket: str) -> S3StorageClass:
+    async def _get_storage_class_for_org(self, organization_id: str) -> S3StorageClass:
```

Ruff (0.14.10)

Unused method argument: `organization_id`

(ARG002)